src/python/zquantum/core/measurement.py

```diff
@@ -392,3 +393,115 @@ def convert_bitstring_to_int(bitstring: Iterable[int]) -> int:
             significant (little endian).
     """
     return int("".join(str(bit) for bit in bitstring[::-1]), 2)
+
+class Measurements:
+    """ A class representing measurements from a quantum circuit. """
+
+    def __init__(self, filename=None):
+        self.counts = {}
+        self.measurements = []
```

Comment:
Actually, perhaps "bitstrings" is better?
Cause measurements.measurements looks a little bit confusing to me ;)

Reply:
Addressed here: 
9e00518